Add a README

Abban opened the pull request:
This contains details of how to install, create, run,
and translate a community banner.

gbirke commented on authors/css/styles.pcss:
```diff
@@ -1,7 +1,7 @@
-	--banner-color-green: #3b9b7d;
-	--banner-color-green-light: #40ad90;
-	--banner-color-green-dark: #25967d;
+	--banner-color-primary: #0d6798;
```
👍

gbirke commented on authors/banner01.js:
```diff
@@ -0,0 +1,17 @@
+	const authorsBanner = new AuthorsBanner( bannerName, bannerTemplate );
+	authorsBanner.init();
+
+	trackingEvents.trackClickEvent( $( '#author-banner-close-button' ), 'banner-closed', bannerCloseTrackRatio );
```
Have you tried if the 2nd event handler for tracking is fired? Because `AuthorsBanner.registerClickEvents` returns `false`, which stops propagation. Also, `AuthorsBanner.registerClickEvents` is still using the "tracking image" technique instead of the EventLoggingTracker.

Please make the EventLoggingTracker part of the AuthorsBanner constructor or `init` function and use it in the event handler.

Abban committed:
Add a README

This contains details of how to install, create, run,
and translate a community banner.

gbirke reviewed (commented):
Thank you!
